pyproject.toml

```diff
@@ -1,17 +1,70 @@
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools>=45",
-    "setuptools_scm>=6.2",
+    "setuptools",
+    "setuptools-scm",
     "wheel",
 ]
 
+[project]
+authors = [
+    {email = "a.lowe@ucl.ac.uk", name = "Alan R. Lowe"},
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: napari",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python",
+    "Topic :: Software Development :: Testing",
```

Comment:
it's not really a testing framework - [napari has](https://github.com/napari/napari/blob/main/setup.cfg#L29):
```
Topic :: Scientific/Engineering :: Visualization
``` 
which might be a better fit

Reply:
Sounds sensible